.gitignore

```diff
@@ -31,3 +31,6 @@ build/
 
 ### VS Code ###
 .vscode/
+
+### Propertis ###
+src/main/resources/liquibase.properties
```

Comment:
I was attempting to do a rollback but it wasn't working without a property file. Therefore, I added the required file to the repository. However, there were login and password inside, and I did not want to risk pushing them to GitHub. To avoid any potential security issues, I decided to add the file to the .gitignore list

I have just created this file.

Reply:
Does your app work without this file at all? Because I think it should. Have you tried specifying all of the necessary variables in `application.properties` instead of this file? Let's invest some time and try to remove it from the project so that our app would still work